templates/terraform/unordered_list_customize_diff.erb

```diff
@@ -0,0 +1,37 @@
+keys := diff.GetChangedKeysPrefix(<%= go_literal(Google::StringUtils.underscore(prop.name)) -%>)
+if len(keys) > 0 {
+    oldCount, newCount := diff.GetChange("<%= Google::StringUtils.underscore(prop.name) -%>.#")
+    var count int
+    // There could be duplicates - worth continuing even if the counts are unequal.
```

Comment:
Shouldn't this be a diff though?
```
Old  New
-a   -a
-a   -b
-b
```

Reply:
Things marked as unordered lists are all intended to be changed to sets in 2.0.0 - it's better to not have user-visible changes when that happens.  So I think this should not be a diff!  :)